src/transaction/flashback.c

```diff
@@ -692,28 +693,26 @@ flashback_find_start_lsa (THREAD_ENTRY * thread_p, FLASHBACK_LOGINFO_CONTEXT * c
   assert (!LSA_ISNULL (&context->end_lsa));
   LSA_COPY (&process_lsa, &context->end_lsa);
 
-  /* fetch log page */
-  error = logpb_fetch_page (thread_p, &process_lsa, LOG_CS_SAFE_READER, log_page_p);
-  if (error != NO_ERROR)
-    {
-      logpb_fatal_error (thread_p, false, ARG_FILE_LINE, "flashback_make_loginfo");
-      goto error;
-    }
-
   while (!LSA_ISNULL (&process_lsa))
     {
       /* fetch page from archive or active, if previous page is needed */
       if (log_page_p->hdr.logical_pageid != process_lsa.pageid)
```

Comment:
`log_page_p->hdr.logical_pageid` seems uninitialized.

Reply:
yes, there can be a trash value in `log_page_p->hdr.logical_pageid`.
I will initialize it to -1. Thanks